core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -2497,6 +2504,15 @@ public RelBuilder sortLimit(int offset, int fetch, RexNode... nodes) {
     return sortLimit(offset, fetch, ImmutableList.copyOf(nodes));
   }
 
+  /** Creates a {@link Sort} by specifying collations.
+   */
+  public RelBuilder sort(RelCollation collation) {
+    final RelNode sort =
```

Comment:
Why not also keep offset and fetch as parameters ?

Reply:
This is a sort, not limit operator. So we don't need them.